### PR TITLE
user controller

### PR DESCRIPTION
**Pull request by AbramovaN**

### CODE REVIEW

File: src/main/java/com/abramova/test1/entity/User.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    private String name;
+    @Column(nullable = false)
+    private String email;
+    @Column(nullable = false)
+    private String password;
+    @Column(name = "is_admin", nullable = false)
+    @Type(type = "org.hibernate.type.NumericBooleanType")
+    private boolean isAdmin;
+
+    public User() {
```
Comment (DarkMerge) [inline src/main/java/com/abramova/test1/entity/User.java]: Duplicated `public User()`

File: src/main/java/com/abramova/test1/controller/UserController.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+    @GetMapping
+    private Iterable<User> list() {
+        return  userRepository.findAll();
+    }
+
+    @GetMapping("{id}")
+    private Optional<User> list(@PathVariable Integer id) {
+        return userRepository.findById(id);
+
```
Comment (DarkMerge) [inline src/main/java/com/abramova/test1/controller/UserController.java]: try to keep your code clean, remove empty blank lines.
good way: 1 blank line between code blocks

File: src/main/java/com/abramova/test1/Test1Application.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package com.abramova.test1;
```
Comment (DarkMerge) [inline src/main/java/com/abramova/test1/Test1Application.java]: please, rename you application from test to normal name, for example: blog-cms

**Review by DarkMerge — APPROVED** [review]

File: src/main/java/com/abramova/test1/entity/User.java (hunk 1 above)
Comment (AbramovaN) [reply src/main/java/com/abramova/test1/entity/User.java]: Empty constructor needed for work

File: src/main/java/com/abramova/test1/controller/UserController.java (hunk 2 above)
Comment (AbramovaN) [reply src/main/java/com/abramova/test1/controller/UserController.java]: Done

File: src/main/java/com/abramova/test1/Test1Application.java (hunk 3 above)
Comment (AbramovaN) [reply src/main/java/com/abramova/test1/Test1Application.java]: Done

**Pull request merged**